### PR TITLE
Run MetaDataExporterTest case in parallel

### PR DESCRIPTION
**Pull request by timowest**

### CODE REVIEW

File: querydsl-core/src/test/java/com/querydsl/core/testutil/Parallelized.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    public Parallelized(Class<?> klass) throws Throwable {
+        super(klass);
+        setScheduler(new ThreadPoolScheduler());
+    }
+
+    private static class ThreadPoolScheduler implements RunnerScheduler {
+        private ExecutorService executor;
+
+        public ThreadPoolScheduler() {
+            int numThreads = Runtime.getRuntime().availableProcessors();
```
Comment (Shredder121) [inline querydsl-core/src/test/java/com/querydsl/core/testutil/Parallelized.java]: Did you test how well this works?
I have 8 available processors, did you test it in some way?
Are all involved elements either local to the thread or threadsafe?

Comment (timowest) [inline querydsl-core/src/test/java/com/querydsl/core/testutil/Parallelized.java]: I tested it with 4 and it works well.
I am not sure if DatabaseMetadata is guaranteed to be threadsafe, but it seems to work here.
Without parallelization it task 1min 23secs for me and parallelized 1min.

Comment (Shredder121) [inline querydsl-core/src/test/java/com/querydsl/core/testutil/Parallelized.java]: The time is not that important to me, but how flaky it is / how much it fails.
If it works well with 4 then it should work well with 8.

Comment (timowest) [inline querydsl-core/src/test/java/com/querydsl/core/testutil/Parallelized.java]: I tried it also with a ForkJoinPool, but the time was the time.

File: querydsl-sql-codegen/src/test/java/com/querydsl/sql/codegen/MetaDataExporterAllTest.java [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+    public static void tearDownClass() throws SQLException {
+        connection.close();
+    }
+
+    @Parameterized.Parameters
+    public static List<Object[]> parameters() {
+        List<Object[]> params = Lists.newArrayList();
+
+        List<NamingStrategy> ns = Arrays.<NamingStrategy>asList(
+                new DefaultNamingStrategy(), new OriginalNamingStrategy());
```
Comment (Shredder121) [inline querydsl-sql-codegen/src/test/java/com/querydsl/sql/codegen/MetaDataExporterAllTest.java]: I find that this reads a little nicer.
```java
        List<? extends NamingStrategy> ns = Arrays.asList(
                new DefaultNamingStrategy(), new OriginalNamingStrategy());

```

**Comment by Shredder121** [comment]
But of course it's not necessary.
Looks good to me.

**Comment by timowest** [comment]
Sorry, I can still do the change.

**Comment by johnktims** [comment]
Looks good.

**Pull request merged**